### PR TITLE
Add ert-storage, clean experiment and webviz-ert to spe1-README

### LINKED ISSUE #1736: Update SPE1 example description
There is a getting started description in the SPE1 example for ERT3. This probably needs some updates now as we are using the new storage instance...

**Definition of done**
The description can be followed and allows you to run the setup using a vanilla installation of ERT3.

### PR DESCRIPTION
**Pull request by ManInFez**
**Issue**
Resolves #1736

### CODE REVIEW

File: ert3_examples/spe1/README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ that highlights the features of ert3 in a reservoir context.
    suffice in bash.
  - ecl2df: In addition to the ERT dependencies the job `summary2json` utilises
    ecl2df. It can be installed via `pip install ecl2df`.
+ - ert-storage: To store results, ert3 utilizes a service called ert-storage.
+   It can be installed via `pip install ert[storage]`.
```
Comment (xjules) [inline ert3_examples/spe1/README.md:17]: or `pip install ert-storage` ?

File: ert3_examples/spe1/README.md [hunk 2]
```diff
@@ -101,18 +109,27 @@ realisations. An important difference is that the ensemble size is no longer
...
+To see the status of all the experiments in the workspace, run `ert3 status`.
+
+#### Visualize experiments
+After experiments have been evaluated. The records labled as output in the
+ensemble-config can be visualized through the visualization solution, `webviz-ert`.
+To launch the visualization tool run `ert vis`.
+
 ### Some additional notes
 ERT3 is highly experimental software in the sense that we aim at moving fast.
 Natural features that is lacking is among others:
```
Comment (xjules) [inline ert3_examples/spe1/README.md:124]: Natural features that `are lacking` or `just lack`

File: ert3_examples/spe1/README.md [hunk 3]
```diff
@@ -101,18 +109,27 @@ realisations. An important difference is that the ensemble size is no longer
...
  - inspect records and experiments in the cli,
- - delete records and experiment runs,
+ - delete workspace records,
  - monitor ongoing experiments,
- - visualize experiment results,
```
Comment (xjules) [inline ert3_examples/spe1/README.md:110]: Is this lacking?

File: ert3_examples/spe1/README.md (hunk 1 above)
Comment (ManInFez) [reply ert3_examples/spe1/README.md:17]: not to get the version currently compliant with the ert version you are installing

File: ert3_examples/spe1/README.md (hunk 3 above)
Comment (ManInFez) [reply ert3_examples/spe1/README.md:110]: nope, not after #1743 is merged, that is why I am removing it.

**Review by xjules — APPROVED** [review]
LGTM! :+1:

**Pull request merged**